src/drivers/ports/esp32/can_port.i

```diff
@@ -176,9 +250,15 @@ static ssize_t write_cb(void *arg_p,
     struct can_driver_t *self_p;
     struct can_device_t *dev_p;
     const struct can_frame_t *frame_p = (struct can_frame_t *)buf_p;
+    ssize_t res;
 
     self_p = arg_p;
     dev_p = self_p->dev_p;
+    
+    /* Return error if CAN-module is waiting to be idle again. */
+    if ( is_waiting_for_bus_idle(self_p) ) {
```

Comment:
Remove spaces, as in ``if (is_waiting_for_bus_idle(self_p)) {``. Make the same change elsewhere in the file.